### ISSUE TITLE
v4.44.1 is bundled with .git. This is causing EISGIT - Appears to be a git repo or submodule.

### ISSUE THREAD

**Issue by achoppam** [sequelize/sequelize#11143]
## What are you doing?
Upgraded package to 4.44.1

**To Reproduce**
1. Upgrade to 4.44.1
2. Do multiple npm install

## What do you expect to happen?
npm install should be fine.

## What is actually happening?
```
This issue happens 4.44.1

npm ERR! path <project-dir>/node_modules/sequelize
npm ERR! code EISGIT
npm ERR! git <project-dir>/node_modules/sequelize: Appears to be a git repo or submodule.
npm ERR! git <project-dir>/node_modules/sequelize
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.

npm ERR! A complete log of this run can be found in:
npm ERR! /users/asciuser/.npm/_logs/2019-07-03T00_32_38_362Z-debug.log
```

## What do you expect to happen?
NPM bundle has .git file/folder too. That is causing this.

Dialect **library** version: NA
Database version: NA
Sequelize version: 4.44.1
Node Version: v10.1.0 (Even in other versions too)
OS: Any
Tested with latest release:
- [ ] No
- [x] Yes, specify that version: Yes. 4.44.1

**Comment by sushantdhiman** [comment]
Fixed by `+ sequelize@4.44.2`

**Comment by andreassiegel** [comment]
Thanks a lot for this quick fix!
